src/popup/router/index.ts

```diff
@@ -70,10 +70,19 @@ router.beforeEach(async (to, from, next) => {
     return;
   }
 
-  if (to.name === ROUTE_APPS_BROWSER && activeAccount.value.protocol !== PROTOCOL_AETERNITY) {
-    setActiveAccountByIdx(0);
-    next({ name: ROUTE_APPS_BROWSER });
-    return;
+  if (to.name === ROUTE_APPS_BROWSER) {
```

Comment:
It can be done by having mobile only list of routes.
```
const router = new Router({
    mode: process.env.VUE_APP_CORDOVA ? 'hash' : 'history',
    scrollBehavior(to, from, savedPosition) {
      if (savedPosition) {
        return savedPosition;
      }
      return { x: 0, y: 0 };
    },
    routes: (await Promise.all([
      import(/* webpackChunkName: "ui-common" */ './routes/common'),
      ENV_MOBILE_DEVICE
        ? import(/* webpackChunkName: "ui-mobile" */ './routes/mobile')
        : undefined
    ])).reduce((p, module) => [...p, ...module.default], []),
  });
  ```

Reply:
we can add it to backlog as an improvement